jep/0000/README.adoc

```diff
@@ -0,0 +1,285 @@
+* Participate in release planning meetings and retrospectives, and burndown
+  meetings, as needed.
+* Ensure related work happens in a project-owned GitHub org and repository,
+  with code and tests explicitly owned and supported by the SIG, including issue
+  triage, PR reviews, test-failure response, bug fixes, etc.
+* Use the above forums as the primary means of working, communicating, and
+  collaborating, as opposed to private emails and meetings
+
+
+All Special Interest Groups are subject to the Jenkins project
+link:https://jenkins.io/conduct[Code of Conduct].
+
+=== Roles
+
+- **SIG Participant**: active in one or more areas of the project; wide
+  variety of roles are represented
+- **SIG Lead**: SIG organizer
+
+=== Creation and Maintenance
+
+==== Prerequisites
+
+* Propose the new SIG publicly, including a brief mission statement, by
+  emailing jenkinsci-dev@googlegroups.com and jenkinsci-users@googlegroups.com,
+  then wait a couple of days for feedback
+* Organize meetings using video or text as needed. No need to wait for the a
+  regularly scheduled Jenkins Governance meeting to discuss. Please report
+  summary of ad hoc SIG meetings to the the SIG mailing list.
+* Use existing proposal and <<pull-request, PR process>>
+* Announce new SIG on jenkinsci-dev@googlegroups.com
+* Submit a PR to add a row for the SIG to the table in the kubernetes/community README.md file, to create a kubernetes/community directory, and to add any SIG-related docs, schedules, roadmaps, etc. to your new kubernetes/community/SIG-foo directory.
+
+=== Creating service accounts for the SIG
+
+With a purpose to distribute the channels of notification and discussion of the
+various topics, every SIG has to use multiple accounts to GitHub mentioning and
+notifications. Below the procedure is explained step-by-step.
+
+NOTE: This procedure is managed and maintained by **[@idvoretskyi](https://github.com/idvoretskyi)**; please, reach him directly in case of any questions/suggestions.
+
+==== Google Groups creation
+
+Create Google Groups at [https://groups.google.com/forum/#!creategroup](https://groups.google.com/forum/#!creategroup), following the procedure:
+
+* Each SIG should have one discussion groups, and a number of groups for mirroring relevant github notifications;
+* Create groups using the name conventions below;
+* Groups should be created as e-mail lists with at least three owners
+  (including tyler at monkeypox.org and verninol at gmail.com to ensure SIG
+  continuity);
+* To add the owners, visit the Group Settings (drop-down menu on the right
+  side), select Direct Add Members on the left side and add Sarah and Ihor via
+  email address (with a suitable welcome message); in Members/All Members
+  select Ihor and Sarah and assign to an "owner role";
+* Set "View topics", "Post", "Join the Group" permissions to be "Public"
+
+Name convention:
+
+* jenkinsci-foo (the discussion group)
```

Comment:
I'm aware this would be inconsistent with historical naming, but should we take opportunity to remove the `ci` suffix from `jenkinsci`?